modules/mobile/app/services/mobile/v0/vaccinations/service.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+module Mobile
+  module V0
+    module Vaccinations
+      # Service that connects to CDC's vaccinations XML table
+      # https://www2.cdc.gov/vaccines/iis/iisstandards/XML.asp?rpt=vax2vg
+      #
+      class Service
+        doc = Nokogiri::XML(URI.open('https://www2.cdc.gov/vaccines/iis/iisstandards/XML.asp?rpt=vax2vg')) do |config|
+          config.strict.noblanks
+        end
+
+        doc.root.children.each do |node|
```

Comment:
I wasn't able to figure out how to fetch this data purely by xpath.

Reply:
I think this way fine, it's easy to read 👍